File: templates/module.py
'''
Serverity: 
Description: 
Author: 
'''

detectVariables = ['']
blockedVariables = ['']
linesCount = 0

from utils.showIssue import showIssue

def vScan(filePath):
    global linesCount

    fileLines = open(filePath , 'r').readlines()
    for singleLine in fileLines:
        linesCount += 1
        singleLine = singleLine.rstrip('\n')

        '''
        scan code here
        '''

    linesCount = 0

File: templates/module.php
<?php
error_reporting(0);
include('utils/php/outputFunction.php');

$linesCount = 0;
$issuesFound = False;

$filePath = $argv[1];
$fileLines = fopen($filePath , "r");

while (( !feof($fileLines) )) {
    $line = fgets($fileLines);
    $line = preg_replace('~[\r\n]+~', '', $line);
    $linesCount++;

    /*
    your scan code here
    */
}

fclose($fileLines);
if (!$issuesFound) { returnFalse(); }
?>
